cypress/integration/scheduled-events-table.spec.ts

```diff
@@ -71,6 +71,52 @@ context('Scheduled events table tests', () => {
         'contain',
         'User operations'
       );
+
+      cy.contains(currentHourDateTime)
+        .parent()
+        .should('have.css', 'backgroundColor', 'rgb(145, 70, 175)');
+      cy.contains(currentHourDateTime)
+        .parent()
+        .should('have.css', 'opacity', '0.6');
```

Comment:
I think, maybe it would be nicer to check for some HTML attribute the class on the element rather than a specific color with opacity because the test should pass in case we decided to change the theme color for the site.